templates/hash/msg.go

```diff
@@ -1,7 +1,11 @@
 package hash
 
-const msgTpl = `
+const hashTpl = `
 // Hash function
+//
+// Deprecated: due to hashing implemention only using field values. The omission
+// of the field name in the hash calculation can lead to hash collisions.
+// Prefer the HashUnique function instead.
 func (m {{ (msgTyp .).Pointer }}) Hash(hasher hash.Hash64) (uint64, error) {
```

Comment:
Does this comment lead to IDE's knowing the function is deprecated? I suppose what I'm asking is, is the comment here preserved?